diff --git a/src/lib/emile/emile_cipher_gnutls.c 
b/src/lib/emile/emile_cipher_gnutls.c
index 491ba78..e703793 100644
--- a/src/lib/emile/emile_cipher_gnutls.c
+++ b/src/lib/emile/emile_cipher_gnutls.c
@@ -384,7 +384,6 @@ emile_cipher_server_connect(Emile_Cipher_Type t)
       case EMILE_TLSv1:
          priority = "NORMAL:%VERIFY_ALLOW_X509_V1_CA_CRT:!VERS-SSL3.0";
          break;
-      case EMILE_SSLv3:
       default:
          return NULL;
      }
